include/dem/dem.h

```diff
@@ -261,6 +277,7 @@ class DEMSolver
   ParticlePointLineFineSearch<dim>     particle_point_line_fine_search_object;
   ParticlePointLineForce<dim>          particle_point_line_contact_force_object;
   std::shared_ptr<Integrator<dim>>     integrator_object;
+  std::shared_ptr<Insertion<dim>>      insertion_object;
   std::shared_ptr<PPContactForce<dim>> pp_contact_force_object;
   std::shared_ptr<PWContactForce<dim>> pw_contact_force_object;
   PVDHandler                           pvdhandler;
```

Comment:
I am not sure the suffix object (*_object) adds anything to the understanding, but if you think this helps.